ui/src/app/applications/components/application-details/application-details.scss

```diff
@@ -246,6 +246,8 @@ $header: 120px;
             &.group-nodes-button-on {
                 color: $argo-color-gray-1;
                 background-color: $argo-color-gray-6;
+                border: 3px solid $argo-color-teal-4;
```

Comment:
Subtle behaviour change is that when this button is clicked and pressed, the toolbar increases in height.  When it is unclicked, the toolbar decreases in height, back to normal.  I think toolbars should remain fixed in size.

Reply:
I choose the position of `graph-options-panel` as `absolute` which keeps the size fixed, however, I still see the slight flickering. I would try to improve it in my follow-up PR

Reply:
Ok, sounds good. Let's leave the unresolved comments as is so we can refer back to them in your follow-up PR.